code/src/components/DirectionButtons.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import styled from "styled-components";
+import { useSelector, useDispatch } from "react-redux";
+import gameplay, { generateNextQuestion } from "reducers/gameplay";
+import loading from "reducers/loading";
+
+const Container = styled.div`
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+`;
+
+const Button = styled.button`
+  justify-self: center;
+  min-width: 75px;
+  padding: 7px;
+  margin: 4px 0;
+  color: black;
+  background-color: white;
+  border: none;
+  font-family: inherit;
+  cursor: pointer;
+  transition: 0.2s ease-in;
+
+  @media (min-width: 768px) {
+    &:enabled:hover {
+      color: white;
+      background-color: tomato;
+    }
+  }
+
+  &:disabled {
+    color: grey;
+    opacity: 0.7;
+  }
+`;
+
+const DirectionButtons = () => {
+  const availableMoves = useSelector((state) =>
+    state.gameplay.question.actions?.map((item) => item.direction)
+  );
+  const dispatch = useDispatch();
+
+  const direction = {
+    north: "North",
+    west: "West",
+    east: "East",
+    south: "South",
+  };
```

Comment:
I like the logic here with the buttons and simplifies readability with having it stored as an object as well